### PR TITLE
update @edoardottt and template

### PR DESCRIPTION
**Pull request by edoardottt**
Add:

- scilla (https://github.com/edoardottt/scilla)
- cariddi (https://github.com/edoardottt/cariddi)

Modified:

- mdx template

### CODE REVIEW

File: projects/_template.mdx [hunk 1]
```diff
@@ -26,6 +27,11 @@ tags: # A list of keywords relevant to the contributors for this project
   - Data Visualization
   - Education
   - Health
+  - Security
+  - Recon
+  - Hacking
+  - Infosec
+  - Pentesting
```
Comment (deammer) [inline projects/_template.mdx:34]: **nitpick:** let's keep this list shorter for now, and if more projects suggest the same tags, we'll update the template.
```suggestion
  - Infosec
```

File: projects/_template.mdx [hunk 2]
```diff
@@ -42,6 +48,7 @@ currentlySeeking: # (optional) These fields are searchable and very helpful for
   - Code Reviewers
   - Technical Writers
   - Mentors
+  - Penetration testers
```
Comment (deammer) [inline projects/_template.mdx:51]: **nitpick:** in the same vein, let's make this a little more generic for now:
```suggestion
  - Testers
```

File: projects/edoardottt/cariddi.mdx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+repoUrl: https://github.com/edoardottt/cariddi
+description: Take a list of domains, crawl urls and scan for endpoints, secrets, api keys, file extensions, tokens and more...
+languages: 
+  - Go
+tags: # A list of keywords relevant to the contributors for this project
+  - Security
+  - Recon
+  - Hacking
+  - Infosec
+  - Pentesting
```
Comment (deammer) [inline projects/edoardottt/cariddi.mdx:12]: **nitpick:** same as above:
```suggestion
  - Infosec
```

File: projects/edoardottt/cariddi.mdx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+avatar: cariddi.png
+websiteUrl: https://www.edoardoottavianelli.it
+twitterUrl: https://twitter.com/edoardottt2
+currentlySeeking: #
+  - Developers
+  - Backend Developers
+  - Penetration testers
```
Comment (deammer) [inline projects/edoardottt/cariddi.mdx:19]: **nipick:** same as above:
```suggestion
  - Developers
  - Backend Developers
  - Testers
```

File: projects/edoardottt/scilla.mdx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+repoUrl: https://github.com/edoardottt/scilla
+description: Information Gathering tool - DNS / Subdomains / Ports / Directories enumeration
+languages: # A list of programming languages used in this project
+  - Go
+tags: # A list of keywords relevant to the contributors for this project
+  - Security
+  - Recon
+  - Hacking
+  - Infosec
+  - Pentesting
```
Comment (deammer) [inline projects/edoardottt/scilla.mdx:12]: **nitpick:** same as above:
```suggestion
  - Infosec
```

File: projects/edoardottt/scilla.mdx [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+avatar: scilla.png
+websiteUrl: https://www.edoardoottavianelli.it
+twitterUrl: https://twitter.com/edoardottt2
+currentlySeeking: #
+  - Developers
+  - Backend Developers
+  - Penetration testers
```
Comment (deammer) [inline projects/edoardottt/scilla.mdx:19]: **nitpick:** same as above (last one!):
```suggestion
  - Developers
  - Backend Developers
  - Testers
```

**Review by deammer — COMMENTED** [review]
Hi @edoardottt, thank you for contributing to OSS Port! Yours are the first Go projects on our site, which is very exciting!

In order to keep the list of tags manageable, I've suggested some adjustments to the ones added in your PR (see our [tag policy](https://github.com/codesee-io/oss-port#tag-policy)).

Additionally, thank you for adding pictures for your projects. I noticed in the [deploy preview](https://deploy-preview-51--oss-port.netlify.app) that, because the images aren't squares, they look very tiny. I would recommend using square versions for better readability. I've taken the liberty to generate square images from the ones you provided, so feel free to use those if you'd like.

|Scilla|Cariddi|
|---|---|
|scilla|cariddi|

I can make those changes for you if you'd like, but I didn't want to push to your `main` branch without your permission. Please let me know if I can help in any way.

**Comment by edoardottt** [comment]
I think I've updated everything

**Review by deammer — APPROVED** [review]
Thanks again, @edoardottt! This will go live on oss-port.com in a few minutes 🚀

**Pull request merged**